Libraries/Text/RCTTextAttributes.m

```diff
@@ -133,9 +133,13 @@ - (NSParagraphStyle *)effectiveParagraphStyle
   // Colors
   RCTUIColor *effectiveForegroundColor = self.effectiveForegroundColor; // TODO(OSS Candidate ISS#2710739)
 
+#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
   if (_foregroundColor || !isnan(_opacity)) {
     attributes[NSForegroundColorAttributeName] = effectiveForegroundColor;
   }
+#else
```

Comment:
Thanks for making this fix.

And thanks for adding the comments: for the sake of merges from upstream, I comment all the delta's from upstream with this notation.   Historically the internal bug number referenced in these comments was tracking contributing the entire macOS implementation upstream.   For various reasons we're not going to contribute macOS up to core, but we still use this comment convention to mark the deltas.   I strive to have `// TODO(macOS ISS#2323203)` for single line deltas, and for blocks I mark them `// [TODO(macOS ISS#2323203)` at the beginning and `// ]TODO(macOS ISS#2323203)` at the end.   In JavaScript where eslint can mangle the comments, I sometimes have to use `/*TODO(macOS ISS#2323203)*/` inline.